### PR TITLE
Schema v2

### PR DESCRIPTION
**Pull request by brandonliu**
v2: reformatted with only sponsors as users

### CODE REVIEW

File: app/models/startup.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    last_name: String,
+    bio: String,
+    team_name: String
+});
+
+var teamSchema = new mongoose.Schema({
+    team_name: String,
+    name_of_main_contact: String,
+    contact_email: String,
+    team_members: [],
```
Comment (shivaalroy) [inline app/models/startup.js]: whats the difference between team_members and Team?

File: app/models/startup.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    sector: String,
+    customers: String,
+    problem: String,
+    solution: String,
+    business_model: String,
+    Team: [participantSchema]
```
Comment (shivaalroy) [inline app/models/startup.js]: lowercase "team"

File: app/models/startup.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+"use strict";
```
Comment (shivaalroy) [inline app/models/startup.js]: rename file to team.js

File: app/models/startup.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    customers: String,
+    problem: String,
+    solution: String,
+    business_model: String,
+    Team: [participantSchema]
+});
+
+
+// create the model for startups and expose it to our app
+var team = mongoose.model('team', teamSchema);
```
Comment (shivaalroy) [inline app/models/startup.js]: team -> "Team". model name is capitalized

File: app/models/user.js [hunk 5]
```diff
@@ -9,9 +9,10 @@ var userSchema = mongoose.Schema({
     local: {
         email: String,
         password: String,
-    }
+    },
```
Comment (shivaalroy) [inline app/models/user.js]: you shouldnt have to change this file at all. revert all changes to this file

File: app/models/startup.js (hunk 1 above)
Comment (brandonliu) [inline app/models/startup.js]: Woops, that's a duplicate field. team: [participantSchema] should just be an array of Participants.

File: app/models/team.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    customers: String,
+    problem: String,
+    solution: String,
+    business_model: String,
+    team: [participantSchema]
+});
+
+
+// create the model for startups and expose it to our app
+var team = mongoose.model('Team', teamSchema);
```
Comment (shivaalroy) [inline app/models/team.js]: we should be consistent and do module.exports = mongoose.model('Team'....

Not sure which way is better, but I think for now being consistent is good

Comment (gshubham) [inline app/models/team.js]: ^^Good call! Haha sorry, I still get notifications from this repository for some reason....

Comment (shivaalroy) [inline app/models/team.js]: LOL thanks Shubham ahahahaha. hope things are going well at Affinity!

**Pull request merged**